GCRCatalogs/cosmodc2.py

```diff
@@ -462,12 +477,17 @@ class UMShearCatalog(UMGalaxyCatalog):
     @staticmethod
     def _generate_quantity_modifiers():
         quantity_modifiers = {
+            'ra': 'ra_lensed',
+            'dec': 'dec_lensed',
+            'convergence': 'conv', 
+            'magnification': (
+                _calc_mag,
+                'conv',
+                'shear_1',
+                'shear_2',
+            ),
             'shear_2_treecorr': 'shear_2',
             'shear_2_phosim':   (np.negative, 'shear_2'),
+            'shear_1':   (np.negative, 'shear_1'),
```

Comment:
This seems to break the backward compatibility for the shear catalog reader? (cc @patricialarsen)